### PR TITLE
New components: field + texfield

### PR DESCRIPTION
**Pull request by vvalentin-lucca**
New components:

- [x] field,
- [x] textfield.

(This PR is more readable commit by commit)

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1651

File: packages/scss/src/components/field/index.scss [hunk 1]
```diff
@@ -3,70 +3,78 @@
...
-		&:is([aria-required]) {
-			@include required($label: '.#{$field}-label');
+		&:is(.is-loading, .loading) {
+			@include framedLoading;
 		}
+	}
 
+	.textfield-input,
+	.radiosfield-input,
+	.checkboxes-input {
```
Comment (AlexiaAlarza) [inline packages/scss/src/components/field/index.scss:54]: isn't it "checkboxesfield-input" ?

File: packages/scss/src/components/field/index.scss [hunk 2]
```diff
@@ -3,70 +3,78 @@
...
-			@include state('error', $label: '.#{$field}-label', $input: '.#{$field}-input', $suffix: '.#{$field}-suffix');
+		&:is(.is-invalid, .invalid, .is-error, .error) {
+			@include inputState('error');
 		}
 	}
 }
+
+.textfield-input,
+.radiosfield-input,
+.checkboxes-input {
```
Comment (AlexiaAlarza) [inline packages/scss/src/components/field/index.scss:75]: idem

File: packages/scss/src/components/field/mods.scss [hunk 3]
```diff
@@ -1,19 +1,177 @@
...
+	flex-direction: row;
+	margin-bottom: 1rem;
+	width: calc(
+		var(--components-field-sizes-default) + var(--components-field-compact-label-right-margin) +
+			var(--components-field-compact-label-sizes-default)
+	);
+
+	.textfield-label,
+	.radiosfield-label,
+	.checkboxes-label {
```
Comment (AlexiaAlarza) [inline packages/scss/src/components/field/mods.scss:23]: checkboxesfield-label ?

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1651

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1651

**Pull request merged**